### PR TITLE
Added followup changes to thread previews

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8069

## Description of Changes
- The "comments" button on any thread card now takes user to the first comment on the thread details page
- Recent comments are no longer shown with thread cards on the community discussion page
- Thread content preview on thread card now correctly applies gradient on the last line of the truncated content (the content with "show more" button)

## "How We Fixed It"
N/A

## Test Plan
- Click on "comments" button on any thread card (you can find thread cards in the community overview, discussions or homepage for-you/global sections), verify it takes you to thread details page and scrolls you to first comment
- Verify that the truncated text (any text with "show more" button) is faded with a slight gray color in thread card
- Recent comments are now removed from the thread card display on community discussions page

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.scss [hunk 1]
```diff
@@ -3,4 +3,9 @@
 .CommentsTree {
   display: flex;
   flex-direction: column;
+  // breadcrumbs don't have an underlying elemnet that would fill in the space it overlays
+  // this causes an issue (first commnet is cutoff by the breadcrumbs overlay) when scrolling
+  // comments tree into view
+  // TODO: fix breadcrumbs and remove this
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.scss:9]: Would be creating a ticket for this before merging this pr

File: packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx [hunk 2]
```diff
@@ -64,6 +64,29 @@ export const CommentTree = ({
...
+          block: 'start',
+        });
+      }, 100);
+    }
+
+    return () => {
+      timeout !== undefined && clearTimeout(timeout);
+    };
+    // we only want to run this on first mount
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx:87]: focusCommentsParam won't change over the time so passing it to the dependency array should be safe

**Review by masvelio — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx (hunk 2 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx:87]: actually, there are some manual redraws that are triggered on this page by unrelated logic to this comment focus. Adding to the dep array causes it to run on redraws which is prevented by running on mount.

File: packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.scss (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.scss:9]: https://github.com/hicommonwealth/commonwealth/issues/8128

**Review by Israellund — APPROVED** [review]

**Pull request merged**